### ISSUE TITLE
Example won't compile because of broken del-msh dependency

### ISSUE THREAD

**Issue by rsaarelm** [kvark/blade#68]
```
$ cargo run --release --example bunnymark
   Compiling del-msh v0.1.17
   Compiling ron v0.8.1
   Compiling egui-gizmo v0.12.0
   Compiling egui_plot v0.23.0
error[E0425]: cannot find function `nearest_to_point3_` in module `tri3`
   --> /home/rsaarelm/.cargo/registry/src/index.crates.io-6f17d22bba15001f/del-msh-0.1.17/src/trimesh3_search.rs:55:40
    |
55  |               let (pn, _r0, _r1) = tri3::nearest_to_point3_(
    |                                          ^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `nearest_to_point3`
    |
   ::: /home/rsaarelm/.cargo/registry/src/index.crates.io-6f17d22bba15001f/del-geo-0.1.14/src/tri3.rs:197:1
    |
197 | / pub fn nearest_to_point3<T>(
198 | |     q0: &nalgebra::Vector3<T>,
199 | |     q1: &nalgebra::Vector3<T>,
200 | |     q2: &nalgebra::Vector3<T>,
...   |
203 | |     where T: nalgebra::RealField + Copy + 'static,
204 | |           f64: AsPrimitive<T>
    | |_____________________________- similarly named function `nearest_to_point3` defined here

error[E0277]: the size for values of type `[_]` cannot be known at compilation time
  --> /home/rsaarelm/.cargo/registry/src/index.crates.io-6f17d22bba15001f/del-msh-0.1.17/src/trimesh3_search.rs:55:18
   |
55 |             let (pn, _r0, _r1) = tri3::nearest_to_point3_(
   |                  ^^ doesn't have a size known at compile-time
   |
   = help: the trait `Sized` is not implemented for `[_]`
   = note: all local variables must have a statically known size
   = help: unsized locals are gated as an unstable feature

Some errors have detailed explanations: E0277, E0425.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `del-msh` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...

$ rustc --version
rustc 1.74.0 (79e9716c9 2023-11-13) (built from a source tarball)
$ git rev-parse HEAD
53d8e88763d529edb343a081e530e1661901a4e8
```

**Comment by kvark** [comment]
See https://github.com/nobuyuki83/del-msh/issues/1

**Comment by kvark** [comment]
Fixed by https://github.com/kvark/blade/commit/daec7a85ff43f737dea4abeacd50825edfc6440e
hopefully del-msh isn't going to break the API in the future...

**Comment by kpwebb** [comment]
Hey just checking on this again: I just tried to build the init, particle and bunnymark examples and got function signature error with `del-msh = "=0.1.17" (checked in daec7a85ff43f737dea4abeacd50825edfc6440e) and the current `del-msh = "=0.1.24"` 

I had to fall back to 0.1.12 to get anything to build. Curious if the del-msh dependency needs to fall back/pin 0.1.12 for the time being?

**Comment by kpwebb** [comment]
Alternatively, if it's only used in the raytracing example could this be moved to a feature dependency for that test?

**Comment by kvark** [comment]
Thanks for the heads up, @kpwebb ! Trying a fix in #85 
I think we need to move away from del-msh entirely instead of dealing with this problematic dependency
